### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by rollerchloe**
Mainly editing the content, so that this readme can be retrieved by documentation website as a remote file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -247,40 +280,36 @@ await tokenRegistry.setRoleAdmin(roleHash.RestorerRole, roleHash.RestorerAdminRo
...
-Can only be called by default admin.
-
 # Deployment
 
-Hardhat is used to manage the contract development environment and deployment. This repository provides a couple of
-Hardhat tasks to simplify the deployment process.
+With Hardhat, you can manage the contract development environment and deployment. This repository provides a couple of Hardhat tasks to simplify the deployment process.
 
-Starting from v4, we have included an easy and cost-effective way to deploy the contracts while also keeping options available for advanced users to setup the contracts their preferred way.
+Starting from V4, OpenAttestation includes an easy and cost-effective way to deploy the contracts and meanwhile keep the options available for advanced users to set up the contracts in a preferable way.
```
Comment (yapyuyou) [inline README.md:287]: The V4 here is referring to the token registry (this repo) specifically, not OpenAttestation. We can change this to "Starting from v4, the token registry..."

File: README.md [hunk 2]
```diff
@@ -210,33 +237,39 @@ Roles are useful for granting users to access certain functions only. Currently,
 | `AccepterRole` | Able to accept a surrendered token  |
 | `RestorerRole` | Able to restore a surrendered token |
 
-A trusted user can be granted multiple roles by the admin user to perform different operations.
-The following functions can be called on the token contract by the admin user to grant and revoke roles to and from users.
 
-#### Grant a role to a user
+The admin user can grant a trusted user multiple roles to perform different operations.
+
+To grant roles to the users or revoke roles from them, the admin user can call on the token contract the following functions:
```
Comment (yapyuyou) [inline README.md:243]:
```suggestion
To grant roles to the users or revoke roles from them, the admin user can call the following functions:
```

File: README.md [hunk 3]
```diff
@@ -388,33 +418,40 @@ DEPLOYER_PK=
...
-Check out our [Token Registry Subgraph](https://github.com/Open-Attestation/token-registry-subgraph) Github repository
+Check out the [Token Registry Subgraph GitHub repository](https://github.com/Open-Attestation/token-registry-subgraph)
 for more information on using and deploying your own subgraphs for the Token Registry contracts.
 
-## Notes
+## Additional information
 
-- The contracts have not gone through formal audits yet. Please use them at your own discretion.
+The contracts have not gone through formal audits. Use them at your own discretion.
+<!--Flag: Question here: Have the contracts gone thru formal audits so far? If not, shall we move this paragraph to the beginning to inform the user before they get started?-->
```
Comment (yapyuyou) [inline README.md:457]: To my knowledge no, but I think we can leave it here at the bottom as we don't want to scare potential users away

**Review by yapyuyou — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (rollerchloe) [reply README.md:243]: removed

**Review by yapyuyou — APPROVED** [review]

**Pull request merged**